keras/backend/tensorflow/optimizer.py

```diff
@@ -97,25 +97,21 @@ def weight_decay_fn(variable):
             variables,
         )
 
-    def _internal_apply_gradients(self, grads_and_vars):
+    def _backend_update_step(self, grads, trainable_variables, learning_rate):
+        trainable_variables = [
+            v.value if isinstance(v, backend.Variable) else v
+            for v in trainable_variables
+        ]
         tf.__internal__.distribute.interim.maybe_merge_call(
-            self._distributed_apply_gradients_fn,
+            self._distributed_tf_update_step,
             self._distribution_strategy,
-            grads_and_vars,
+            list(zip(grads, trainable_variables)),
+            learning_rate,
         )
 
-    def _distributed_apply_gradients_fn(
-        self, distribution, grads_and_vars, **kwargs
+    def _distributed_tf_update_step(
+        self, distribution, grads_and_vars, learning_rate
```

Comment:
Seems that the learning_rate is not used here, the value in the apply_grad_to_updatew_var was retrieved from self._get_current_learning_rate(). Did I miss anything?